pom.xml

```diff
@@ -35,6 +41,15 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+
+			<plugin>
+				<groupId>org.liquibase</groupId>
+				<artifactId>liquibase-maven-plugin</artifactId>
+				<version>4.4.0</version>
```

Comment:
Let's put the liquibase version in the properties section.
https://maven.apache.org/pom.html#Properties
Plugin and dependency versions are highly coupled, so it makes sense to store it at one place

Reply:
Let's put the first liquibase script which will create the database.
You can find the sql here: https://github.com/mtalstykh-exadel/team4_docs/blob/main/architecture/backend/database-sceme/db/language-testing.sql